lib/template/content_response.ex

```diff
@@ -0,0 +1,31 @@
+defmodule SparkPost.Template.ContentResponse do
```

Comment:
Any reason not to name this `SparkPost.Template.Content` ? The form is the same when creating, retrieving and previewing.

Better yet, we could re-use `SparkPost.Content.Inline` since it uses the same model. We could move `convert_from_field` into `SparkPost.Template` or `SparkPost.Content.Inline` as you see fit.